### ISSUE TITLE
"stepcvt" not found as a command

### ISSUE THREAD

**Issue by Asai-Yume** [sree314/stepcvt#67]
Both Yumeng and I are encountering the problem of not being able to run the executable properly, could it be something wrong when installing the dependency on Windows/Mac OS?

**Comment by sree314** [comment]
I don't know what the exact error you're encountering is, but it is almost certainly related to how PATH operates.

(it *should* work on Mac though).

My suggestion is to find the path of the executable relative to the test script, and invoke it as "python stepcvt".

**Comment by Asai-Yume** [comment]
Yumeng is working on a Mac, and she ran into the same issue I did, in which stepcvt is installed under the ./python/scripts folder but not being recognized by PATH. For now I believe we could just manually do something like running "python ./path/to/scripts", but should we take a look at it tomorrow so that we could avoid adding the path everytime?

**Comment by sree314** [comment]
Yes, we can look at it tomorrow. Note that the stepcvt script is in `../scripts/stepcvt` which should be stable relative to the test file.
